provisioning_templates/script/windows_default_script.erb

```diff
@@ -13,6 +13,7 @@ params:
 <%
   proxy_uri = host_param('http-proxy') ? "http://#{host_param('http-proxy')}:#{host_param('http-proxy-port') || 3128}" : nil
   proxy_string = proxy_uri ? "-e http_proxy=#{proxy_uri}" : ''
+  windowsEFI = host_param('windowsEFI', 'false')
```

Comment:
Can you use `host_param_true?` or `_false?` these macros are available. They will parse "true", "yes" and "1" etc.

Reply:
I have updated line 16 - is that now as you suggest?  Thanks.